.gitignore

```diff
@@ -1,7 +1,7 @@
 /bin/
 # Ignoring the lib folder that has all the JAR files, these can be more than 300 mb which wont work on git
 # These jar files are needed for the project and are on the GaudiumLabs google drive
-/lib
+*.jar
```

Comment:
dont ignore the full directory of lib, just the jar files. The directory structure is important as that is how it is imported into the code